compiler/test/dotty/tools/vulpix/TestFlags.scala

```diff
@@ -26,23 +26,17 @@ final case class TestFlags(
   private val languageFeatureFlag = "-language:"
   private def withoutLanguageFeaturesOptions = options.filterNot(_.startsWith(languageFeatureFlag))
 
-  // TODO simplify to add `-language:feature` to `options` once
-  //      https://github.com/lampepfl/dotty-feature-requests/issues/107 is implemented
-  def andLanguageFeature(feature: String) = {
-    val (languageFeatures, rest) = options.partition(_.startsWith(languageFeatureFlag))
-    val existingFeatures = if (languageFeatures.isEmpty) languageFeatures.mkString(",") + "," else ""
-    copy(options = rest ++ Array(languageFeatureFlag + existingFeatures + feature))
-  }
+  def andLanguageFeature(feature: String) =
+    copy(options = options ++ Array(s"$languageFeatureFlag$feature"))
 
-  def withoutLanguageFeature(feature: String) = {
+  def withoutLanguageFeature(feature: String) =
     val (languageFeatures, rest) = options.partition(_.startsWith(languageFeatureFlag))
-    val filteredFeatures = languageFeatures.filter(_ == feature)
```

Comment:
This is that thing where you go, Is it filter-in or filter-out?

A useful tool would find all the boolean expressions and flip their values until the test passes.